src/Collections/Sequences/LittleEndianNat.dfy

```diff
@@ -68,6 +68,7 @@ abstract module LittleEndianNat {
           ToNatLeft(xs);
           Last(xs) * Pow(BASE(), |xs| - 1);
             {
+              reveal Pow();
```

Comment:
Do you need the lemmas below if you reveal Pow?

Reply:
Nope! Good catch.